test/local/routes/password.js

```diff
@@ -317,6 +319,7 @@ describe('/password', () => {
           assert.equal(mockMailer.sendPasswordChangedNotification.getCall(0).args[2].location.city, 'Mountain View')
           assert.equal(mockMailer.sendPasswordChangedNotification.getCall(0).args[2].location.country, 'United States')
           assert.equal(mockMailer.sendPasswordChangedNotification.getCall(0).args[2].timeZone, 'America/Los_Angeles')
+          assert.equal(mockMailer.sendPasswordChangedNotification.getCall(0).args[2].uid, uid)
```

Comment:
Look's like we never had an assert on `/forgot/verify_code` for `mailer.sendPasswordResetNotification`. Could probably add one and check that uid is passed.